hca/util/__init__.py

```diff
@@ -529,14 +530,15 @@ def _process_method_args(self, parameters, body_json_schema):
         return body_props, method_args
 
     def _build_client_method(self, http_method, http_path, method_data):
+        http_path = http_path.replace('.', '').replace('-', '_')
         method_name_parts = [http_method] + [p for p in http_path.split("/")[1:] if not p.startswith("{")]
         method_name = "_".join(method_name_parts)
         if method_name.endswith("s") and (http_method.upper() in {"POST", "PUT"} or http_path.endswith("/{uuid}")):
             method_name = method_name[:-1]
 
         parameters = {p["name"]: p for p in method_data.get("parameters", [])}
         body_json_schema = {"properties": {}}
-        if "requestBody" in method_data:
+        if "requestBody" in method_data and "application/json" in method_data["requestBody"]["content"]:
```

Comment:
Does `method_data["requestBody"]` always contain `["content"]`?

Reply:
Yes it does according [openapi 3.0](https://swagger.io/docs/specification/describing-request-body/)